kairos-cli/src/client.rs

```diff
@@ -0,0 +1,58 @@
+use kairos_server::routes::PayloadBody;
+
+use reqwest::{Client, Url};
+use serde::{Deserialize, Serialize};
+use std::fmt;
+
+#[derive(PartialOrd, Ord, PartialEq, Eq, Debug, Serialize, Deserialize)]
+pub enum KairosClientError {
+    ResponseError(String),
+    ResponseErrorWithCode(u16, String),
+    DecodeError(String),
+    KairosServerError(String),
```

Comment:
Please remove/comment unused variant.

Reply:
It should actually be used when the server returns an error, so it's unused by accident. It's used now here [9d0ae91](https://github.com/cspr-rad/kairos/pull/91/commits/9d0ae91ca85582c932c3a81a8c527628baaf8392)